sopel/config/core_section.py

```diff
@@ -178,6 +178,9 @@ def homedir(self):
     It is a regular expression (so the default, ``\.``, means commands start
     with a period), though using capturing groups will create problems."""
 
+    reply_errors = ValidatedAttribute('reply_errors',bool,default=False)
```

Comment:
That makes sense.

Ah, I didn't realize that was the convention until rereading the other definitions. Thanks.